res/controllers/Pioneer-DDJ-400.midi.xml

```diff
@@ -516,7 +516,7 @@
             <control>
                 <description>RELOOP/EXIT +SHIFT (DECK1) - press - Active loop on/off</description>
                 <group>[Channel1]</group>
```

Comment:
What about the other decks?

Reply:
Changed it for Deck 2 as well.

Reply:
Please update the wiki to reflect this change with `reloop_andstop`.